tensorflow/BUILD

```diff
@@ -469,6 +469,7 @@ cc_binary(
     linkshared = 1,
     deps = [
         "//tensorflow/c:c_api",
+        "//tensorflow/c/eager:c_api",
```

Comment:
I see what the problem is: this line should be added immediately before `//tensorflow/core:tensorflow`, instead of here. Buildifier is complaining about the sort order.